openvdb_houdini/houdini/SOP_OpenVDB_Scatter.cc

```diff
@@ -89,6 +90,19 @@ newSopOperator(OP_OperatorTable* table)
         .setDefault(PRMzeroDefaults)
         .setTooltip("If enabled, the output will contain the input VDB grids."));
 
+    {
+        const char* items[] = {
+            "vdb", "VDB Points",
+            "hdk", "Houdini Points",
+            nullptr
+        };
+
+        parms.add(hutil::ParmFactory(PRM_ORD, "pointtype", "Points To Scatter")
+            .setDefault(PRMoneDefaults)
+            .setChoiceListItems(PRM_CHOICELIST_SINGLE, items)
+            .setTooltip("The type of geometry to scatter."));
```

Comment:
I'd much prefer this to be exposed as a toggle (defaulting to off) "Scatter VDB Points". I think it's highly unlikely we'll ever output another type of points here and a toggle is simpler because it means you don't have to click the drop-down in order to see what types of points you can scatter to.

Reply:
sure